components/canvas/utils/nodeValidityHelper.tsx

```diff
@@ -104,7 +104,10 @@ export const getOptionsAddNode = (node: Node<NodeData>) => {
   if (isChoiceChild) {
     validPositions = {
       ...validPositions,
+      /*
+      Add this back in when the order of choice children has been fixed
```

Comment:
Could be smart to add `TODO` to make it easier for people discovering this in TODO-tools or when searching for TODO. Could also be a good idea to add this as a sub task of an issue.